src/components/banner/Banner.tsx

```diff
@@ -157,9 +157,29 @@ export const Banner = ({
   onClose,
   accessibilityHint,
   accessibilityLabel,
-  testID,
+  testID
 }: Banner) => {
-  const isPressed: Animated.SharedValue<number> = useSharedValue(0);
+  const isPressed: SharedValue<number> = useSharedValue(0);
+
+  const { isExperimental } = useIOExperimentalDesign();
+  const { themeType } = useIOThemeContext();
+  const theme = useIOTheme();
```

Comment:
How about creating an `isDark` constant derived from the `useIOTheme` hook?

Reply:
@LeleDallas It's a great suggestion, and it could make the code clearer and less verbose. But it also requires a bit of refactoring on both the `io-app-design-system` and `io-app` codebases, as the `useIOTheme` only returns a single object and we need to return two things: the `theme` object and the `isDark` boolean value.

I'd open a specific PR just for this. What do you think?

Reply:
Sounds good to me!